app/locales/es/translations.js

```diff
@@ -1,55 +1,94 @@
 export default {
   dashboard: {
-    title: 'Que deseas hacer?'
+    title: '¿Qué deseas hacer?'
```

Comment:
Perhaps use `desea` instead of `deseas`. The former is more formal (`usted` vs `tu`).